### DISCUSSION TITLE
STM32C011F6P6

### DISCUSSION THREAD

**Discussion by Apehaenger** [modm-io/modm#1187] [Q&A] [answered]
Just started with this interesting HAL approach but unfortunately stumble in the beginning.

Tried to get it running for a custom PCB with a STM32C011F6P6.

For sure I saw at the [main page](https://github.com/modm-io/modm#microcontrollers) that's not officially supported yet, but found it listed in the https://github.com/modm-io/modm-devices repository.

When I try `<option name="modm:target">stm32c011f6p6</option>` I get an "Target is unknown".

Stupid newbie failure or does someone has a hint what to do?

**Comment by salkinium** [comment, marked as answer]
Try enabling it here: 
https://github.com/modm-io/modm/blob/develop/repo.lb#L86

There will probably be a bunch of failures in lbuild, try to hack it and open a PR which I can then collaborate on.

**Reply by rleh** [reply]
You could also take a look at other pull requests from the past that added support for new STM32 series to modm, like e.g. STM32U5 in #843.

**Reply by salkinium** [reply]
Good point, here's another older PR for STM32G0, which is probably more similar to STM32C0: https://github.com/modm-io/modm/pull/167

**Comment by Apehaenger** [comment]
Quite thanks for your quick answer and your useful hints.
Will do my best ;-)
